vault/resource_ldap_auth_backend.go

```diff
@@ -193,6 +194,11 @@ func ldapAuthBackendResource() *schema.Resource {
 			Computed:  true,
 			Sensitive: true,
 		},
+		consts.FieldConnectionTimeout: {
+			Type:     schema.TypeString,
```

Comment:
Any reason we aren't using a TypeInt like `vault_ldap_secret_backend`:

https://github.com/hashicorp/terraform-provider-vault/blob/d8d1353ce7dbefb81e03fa670a491f5c2b6b8877/vault/resource_ldap_secret_backend.go#L72-L77

Reply:
Funny--i started that way, but when I ran terraform with the custom built binary I got a panic.  I might have missed a configuration, I'll try it again so I can post any specific errors or issues I run into

Reply:
I missed a configuration--my fault.  Has been updated to `int` and the test should no longer have the `30s` issue.